sql-schema/135.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE state_translations MODIFY state_value INTEGER NOT NULL;
```

Comment:
Yes as the values returned by the Riverbed are bigger than what tinyint can handle.